lib/view_model/send/output.dart

```diff
@@ -126,8 +126,8 @@ abstract class OutputBase with Store {
 
       if (_wallet.type == WalletType.bitcoin) {
         if (_settingsStore.priority[_wallet.type] == bitcoin!.getBitcoinTransactionPriorityCustom()) {
-          fee = bitcoin!.getFeeAmountWithFeeRate(
-              _settingsStore.customBitcoinFeeRate, formattedCryptoAmount, 1, 1);
+          fee = bitcoin!.getFeeAmountWithFeeRate(_wallet,
+              _settingsStore.customBitcoinFeeRate, 1, 0);
```

Comment:
shouldn't the output count be 1 so it's calculated successfully?

also are we calculating the estimated fee based on a single output? what if we have multiple outputs

Reply:
I've updated the code, now the custom fee rate calculation aligns with the default priority flow. It correctly calculates inputsCount based on the amount in both cases. outputsCount depends on this condition: (amount != null ? 2 : 1).